packages/test-harness/src/launchVscodeAndRunTests.ts

```diff
@@ -81,10 +81,7 @@ export async function launchVscodeAndRunTests(extensionTestsPath: string) {
       // don't bother.  Can be re-enabled if we ever need it; on windows it only
       // hangs some of the time, so might be enough to get a crash dump when you
       // need it.
-      launchArgs:
-        useLegacyVscode || os.platform() === "win32"
-          ? undefined
-          : [`--crash-reporter-directory=${crashDir}`, `--logsPath=${logsDir}`],
+      launchArgs: ["--force-node-api-uncaught-exceptions-policy=true"],
```

Comment:
This seems like a good thing but curious what prompted it

Reply:
https://github.com/cursorless-dev/cursorless/actions/runs/14509376075/job/40704611423?pr=2893

Reply:
Looks like this argument isn't available in electron. Electron apparently enforces note options quite strictly.